Sources/web3swift/Utils/ENS/ENS.swift

```diff
@@ -106,9 +106,9 @@ public class ENS {
         guard isAddrSupports else {
             throw Web3Error.processingError(desc: "Address isn't supported")
         }
-        var options = transaction ?? defaultTransaction
-        options.to = resolver.resolverContractAddress
-        guard let result = try? await resolver.setAddress(forNode: node, address: address, transaction: options, password: password) else {
+        var transaction = transaction ?? defaultTransaction
+        transaction.to = resolver.resolverContractAddress
```

Comment:
As a result of fixing merge conflicts, I've added a couple of variable renamings.

Reply:
From `options` to `transaction`.